packages/manager/apps/pci-rancher/tailwind.config.js

```diff
@@ -1,6 +1,9 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  content: [
+    './src/**/*.{js,jsx,ts,tsx}',
+    '../../../manager-components/src/**/*.{js,jsx,ts,tsx}',
```

Comment:
**nitpick:** As `manager-components` package is independent, we cannot force the consumers of the package to include it in their `tailwind.config`.
```suggestion
```